code-reviews/recipes/Terraform.md

```diff
@@ -53,10 +58,16 @@ echo -e "\n\n>>> Terraform validate"
 terraform validate
 ```
 
-> TODO: Add example of how to use this in an AzDO build validation pipeline
-
-## Code Review Checklist
-
+### Code Review Checklist
```

Comment:
NIT: This all looks to be Italic, not sure if this is intentional.

Reply:
@fnocera - It is not in Italic, can you give more details around which line appears in Italic?